yarn.lock

```diff
@@ -15984,7 +15989,7 @@ modify-values@^1.0.0:
   resolved "https://registry.yarnpkg.com/modify-values/-/modify-values-1.0.1.tgz#b3939fa605546474e3e3e3c63d64bd43b4ee6022"
   integrity sha512-xV2bxeN6F7oYjZWTe/YPAy6MN2M+sL4u/Rlm2AHCIVGfo2p1yGmBHQ6vHehl4bRTZBdHu3TSkWdYgkwpYzAGSw==
 
-moment@^2.21.0, moment@^2.22.2, moment@^2.5.1:
+moment@^2.21.0, moment@^2.5.1:
```

Comment:
So `moment` is still in here; is it coming in transitively? What does `yarn why moment` say?

Reply:
It comes from [gatsby](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/package.json#L108).

```
   - "_project_#clayui.com#gatsby" depends on it
   - Hoisted from "_project_#clayui.com#gatsby#moment"
   - Hoisted from "_project_#@clayui#css#metalsmith-permalinks#moment"
```